deliveries/Multix-a-simple-UI-for-complex-multisigs_1.md

```diff
@@ -0,0 +1,27 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [Multix application](https://github.com/w3f/Grants-Program/blob/master/applications/Multix-a-simple-UI-for-complex-multisig.md). 
+* **Milestone Number:** 1
+
+**Context** (optional)
+We have delivered the interface, it's live on Rococo and it can be found here: https://multix.chainsafe.io/
+There are a couple UX improvements to be made in the very short term, notably for the multisig creation flow, those can be found [in the issue tracker](https://github.com/ChainSafe/Multix/issues).
+
+**Deliverables**
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0.b | Documentation | [Multix repo](https://github.com/ChainSafe/Multix) | The code is commented in line and the flows should be self explanatory for users. | 
+| 0.d  | Docker | [Multix indexer](https://github.com/ChainSafe/Multix/tree/main/squid) | The docker of the indexer is available in the repo | 
+| 0.e  | Article | [Medium article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d) | An article explaining what Multix brings and educating about proxies has been published| 
```

Comment:
Add License delivery (and adjust format)
```suggestion
| ------------- | ------------- | ------------- |------------- |
| 0a. | License | [LICENSE](https://github.com/ChainSafe/Multix/blob/main/LICENSE) | Apache 2.0 | 
| 0b. | Documentation | [Multix repo](https://github.com/ChainSafe/Multix) | The code is commented in line and the flows should be self explanatory for users. | 
| 0d.  | Docker | [Multix indexer](https://github.com/ChainSafe/Multix/tree/main/squid) | The docker of the indexer is available in the repo | 
| 0e.  | Article | [Medium article](https://blog.chainsafe.io/multix-a-simple-interface-to-manage-complex-multisigs-on-polkadot-97328be26f9d) | An article explaining what Multix brings and educating about proxies has been published| 
```

Reply:
of course, thanks for suggesting it, makes things much simpler :)